<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EEDInpaintingDialog</class>
 <widget class="QDialog" name="EEDInpaintingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inpainting by Edge Enhancing Anisotropy Diffusion</string>
  </property>
  <widget class="QGroupBox" name="groupBox_eedInpaint">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>50</y>
     <width>400</width>
     <height>380</height>
    </rect>
   </property>
   <property name="title">
    <string>Inputs</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_eedInpaintTol">
          <property name="text">
           <string>Tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eedInpaintTol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_eedInpaintTimeStep">
          <property name="text">
           <string>Time Step Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eedInpaintTimeStep"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eedInpaintExplitScheme">
        <property name="text">
         <string>Explicit Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eedInpaintFED">
        <property name="text">
         <string>Fast Explicit Diffusion (FED) Scheme</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButton_eedInpaintFSI">
        <property name="text">
         <string>Fast Semi_iterative Scheme (FSI)</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_eedInpaintMSETol">
          <property name="text">
           <string>MSE Tolerance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eedInpaintMSETol"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_eedInpaintInnerFEDCycSize">
          <property name="text">
           <string>Inner Cycle Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_eedInpaintInnerFEDCycSize"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_eedInpaintFSILoopSize">
          <property name="text">
           <string>Inner Loop Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_EEDInpaintFSILoopSoze"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eedInpaintUplImg">
        <property name="text">
         <string>Load Image Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eedInpaintUplMask">
        <property name="text">
         <string>Load Mask Data</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_eedInpaintRun">
        <property name="text">
         <string>Run</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_EEDInpaintMonitoring">
        <property name="text">
         <string>Monitoring</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_eedInpaintImg">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>500</width>
     <height>500</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_eedInpaintMaskImg">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>490</y>
     <width>450</width>
     <height>450</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>860</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_eedInpaintFileName">
      <property name="text">
       <string>File Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_eedInpaintFileName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_eedInpaintSaveImg">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_eedIntp_zeros2mask">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>430</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Zeros to Mask</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_load_eigenInfo">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>470</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Load eigen information</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
